lib/widgets/todo_items.dart

```diff
@@ -38,6 +42,13 @@ class _TodoItemsState extends State<TodoItems> {
           DateFormat.yMMMEd().add_jm().format(DateTime.parse(widget.todo.date)),
           style: TextStyle(fontStyle: FontStyle.italic),
         ),
+        controlAffinity: ListTileControlAffinity.leading,
+        onChanged: (bool completedTask) {
+          setState(() {
+            _isCompleted = completedTask;
+          });
+        },
+        tileColor: _isCompleted ? Color.fromRGBO(202, 240, 248, 1) : Color.fromRGBO(144, 224, 239, 1),
```

Comment:
I'm pretty sure you have a class that holds the colors and these colors must be one of them. If none, make one. 🤔